scripts/build_frontend.py

```diff
@@ -0,0 +1,226 @@
+import os
```

Comment:
In the future please use `git mv` when you move files. Now it thinks you deleted and recreated it so I have no way of seeing what changed here.